src/views/search/search.module.css

```
@@ -0,0 +1,180 @@
+
+.error button:hover {
+  color: var(--post-link-text-color-hover);
+}
+
+.results {
+  overflow-x: hidden;
+  overflow-y: hidden;
+}
+
+/* Registered by use-search-match-highlight while the results are on screen. */
+:global(::highlight(search-match)) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)